CONTRIBUTING.md

```diff
@@ -88,4 +89,16 @@ All contributions (including pull requests) must agree to the Developer Certific
 git commit -s -m "adding X to change Y"
 ```
 
+## Release Package
+
+Ensure you have publish rights for the [NPM package](https://www.npmjs.com/package/speedo). You can release new version simply by running a manual action called `Manual NPM Publish`. Remember to set *releaseType* to one of `patch`, `minor` or `major`.
```

Comment:
https://www.npmjs.com/package/speedo to https://www.npmjs.com/package/network-viewer